pentagon/filters.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def register_filters():
+	"""Register a function with decorator"""
+	registry = {}
+	def registrar(func):
+		registry[func.__name__] = func
+		return func 
+	registrar.all = registry
+	return registrar
+
+
+filter = register_filters()
+
+
+def get_jinja_filters():
+	"""Return all registered custom jinja filters"""
+	return filter.all
+
+
+@filter
```

Comment:
I really like the decorator. It will allow components to register custom filters as well.